packages/mdc-typography/_variables.scss

```diff
@@ -106,5 +106,12 @@ $mdc-typography-styles: (
     font-weight: map-get($mdc-typography-font-weight-values, regular),
     letter-spacing: .08em,
     margin: -.5rem 0 1rem -.04em /* -8sp 0 16sp -.04em */
+  ),
+  button: (
+    font-size: .875rem, /* 14sp */
```

Comment:
https://github.com/material-components/material-components-web/blob/master/packages/mdc-button/mdc-button.scss#L42

font-size was purposely set to 14px...but I cannot figure out why it wasn't .0975rem in the first place, so maybe this "fixes" it.